R/eda.R

```diff
@@ -145,11 +147,34 @@ calc_cor <- function(dataframe, num_vars) {
 #'
 #' @param dataframe the dataframe to be inspected.
 #'
-#' @return a list of vectors; each vector corresponds to a column and
-#' each value inside the vector is 0 if the corresponding value is NA,
+#' @return a tibble; each column corresponds to the same column in dataframe
+#'  and each value inside the columnr is 0 if the corresponding value is NA,
 #' 1 otherwise.
-#' TODO write meaningful examples as implementation goes on
+#'
+#' @export
+#'
+#' @examples
+#' df <- data.frame(x = (c(2,3,4)), y= c(1,10,3))
+#' col_num <- describe_na_values(df)
+#'
+#' #> # A tibble: 2 x 3
+#' #>       x     y
+#' #>   <int> <int>
+#' #> 1     1     1
+#' #> 2     1     1
+#' #> 3     1     1
+#'
+#'#' df <- data.frame(x = (c(2,NaN,4)), y= c(1,10,3))
+#' col_num <- describe_na_values(df)
+#'
+#'  #> # A tibble: 2 x 3
+#' #>       x     y
+#' #>   <int> <int>
+#' #> 1     1     1
+#' #> 2     0     1
+#' #> 3     1     1
+#'
 describe_na_values <- function(dataframe) {
-  #TODO implement function
-  list(c(0))
+  ret_dataframe <-purrr::map(dataframe, function(x){as.numeric(!is.na(x))})
```

Comment:
You can write some defensive code to check for the valid input types. This s one of the requirement of this milestone. Accordingly you can write some test cases to test these exceptions